integration-tests/tests/package.json

```diff
@@ -19,7 +19,6 @@
     "realm": "*"
   },
   "devDependencies": {
-    "@thi.ng/bench": "^3.1.16",
```

Comment:
We should either find a different dependency and fix these performance tests, as I imagine we'll rely on them again very soon. Or alternatively fix the Metro config so it can deal with ES6 modules as expected.

Reply:
I'm on it right now.  I just wanted to unblock @RedBeard0531 and wasn't getting any further with this.